Gemfile.lock

```diff
@@ -363,15 +366,15 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.5.0)
+    rubocop-rails (2.5.2)
```

Comment:
### [rubocop-rails](https://docs.rubocop.org/projects/rails/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.5.0 → 2.5.2

[_[change-log](https://github.com/rubocop-hq/rubocop-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop-hq/rubocop-rails/)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/rubocop-hq/rubocop-rails/compare/v2.5.0...v2.5.2).

These are the first 10 commits:
- (0c7b435) [Fix a typo](https://github.com/rubocop-hq/rubocop-rails/commit/0c7b435df2bb9ce339dee997bef96fb9916d2a87)
- (883e1ec) [Merge pull request #212 from d5uke/fix-typo-in-schema-loader](https://github.com/rubocop-hq/rubocop-rails/commit/883e1eccbb45c7a1dd5b36269b6561b1eb0f25f4)
- (bb266fd) [No UniqueValidationWithoutIndex offense if validation has a …](https://github.com/rubocop-hq/rubocop-rails/commit/bb266fd942f756e75209cde0430162659e216e01)
- (bc09d9d) [Merge pull request #213 from sunny/no-offense-for-unique-val…](https://github.com/rubocop-hq/rubocop-rails/commit/bc09d9deba2913d0f9da7603dfe0ec671ddb3cce)
- (e952888) [Tweak a changelog entry](https://github.com/rubocop-hq/rubocop-rails/commit/e952888be1b46b19124ca294b1e272fea22b4548)
- (ac766e3) [[Fix #215] Fix a false positive for `Rails/UniqueValidationW…](https://github.com/rubocop-hq/rubocop-rails/commit/ac766e373647cf9a0e18fce95b14bc81de3ce777)
- (789a8f2) [Merge pull request #217 from koic/fix_a_false_positive_for_u…](https://github.com/rubocop-hq/rubocop-rails/commit/789a8f2dd389a8da13a64f9e2de18f7dd1316530)
- (fdeeba5) [[Fix #214] Fix an error for `Rails/UniqueValidationWithoutIn…](https://github.com/rubocop-hq/rubocop-rails/commit/fdeeba53276e4ea3aac75719980ee47edbb01235)
- (3ba2b30) [Merge pull request #218 from koic/fix_error_for_unique_valid…](https://github.com/rubocop-hq/rubocop-rails/commit/3ba2b3038b2d11383da2b05a5441c07512315add)
- (3fa9c7c) [[Fix #221] Make `Rails/UniqueValidationWithoutIndex` aware o…](https://github.com/rubocop-hq/rubocop-rails/commit/3fa9c7c41f8ba99b102eb505cd7276e3bd4a4280)


</details>